### PR TITLE
Add the Snowtype GTM instructions page

### PR DESCRIPTION
**Pull request by igneel64**

### CODE REVIEW

File: docs/collecting-data/code-generation/working-with-gtm/index.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+---
+title: "Using Snowtype with Google Tag Manager"
+sidebar_position: 6
+---
+
+:::info
+This feature is available from version 0.5.0.
+:::
+
+To make working with Google Tag Manager and event tracking easier, we created a specific target for Snowtype fitting the way Google Tag Manager handles custom JavaScript code.
```
Comment (gibbok-snowplow) [inline docs/collecting-data/code-generation/working-with-gtm/index.md:10]: I would write few use cases/benefits (even as simple as bullet points) about this feature

**Review by gibbok-snowplow — APPROVED** [review]
LGTM, just added a possible improvement, please have a look

File: docs/collecting-data/code-generation/working-with-gtm/index.md (hunk 1 above)
Comment (igneel64) [reply docs/collecting-data/code-generation/working-with-gtm/index.md:10]: Done!

Comment (gibbok-snowplow) [reply docs/collecting-data/code-generation/working-with-gtm/index.md:10]: I like it! Well done!

**Pull request merged**